extensions/merge-conflict/src/commandHandler.ts

```diff
@@ -0,0 +1,280 @@
+			vscode.commands.registerTextEditorCommand('merge-conflict.accept.both', this.acceptBoth, this),
+			vscode.commands.registerTextEditorCommand('merge-conflict.accept.all-current', this.acceptAllCurrent, this),
+			vscode.commands.registerTextEditorCommand('merge-conflict.accept.all-incoming', this.acceptAllIncoming, this),
+			vscode.commands.registerTextEditorCommand('merge-conflict.accept.all-both', this.acceptAllBoth, this),
+			vscode.commands.registerTextEditorCommand('merge-conflict.next', this.navigateNext, this),
+			vscode.commands.registerTextEditorCommand('merge-conflict.previous', this.navigatePrevious, this),
+			vscode.commands.registerTextEditorCommand('merge-conflict.compare', this.compare, this)
+		);
+	}
+
+	acceptCurrent(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, ...args): Promise<void> {
+		return this.accept(interfaces.CommitType.Current, editor, ...args);
+	}
+
+	acceptIncoming(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, ...args): Promise<void> {
+		return this.accept(interfaces.CommitType.Incoming, editor, ...args);
+	}
+
+	acceptBoth(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, ...args): Promise<void> {
+		return this.accept(interfaces.CommitType.Both, editor, ...args);
+	}
+
+	acceptAllCurrent(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, ...args): Promise<void> {
+		return this.acceptAll(interfaces.CommitType.Current, editor);
+	}
+
+	acceptAllIncoming(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, ...args): Promise<void> {
+		return this.acceptAll(interfaces.CommitType.Incoming, editor);
+	}
+
+	acceptAllBoth(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, ...args): Promise<void> {
+		return this.acceptAll(interfaces.CommitType.Both, editor);
+	}
+
+	async compare(editor: vscode.TextEditor, edit: vscode.TextEditorEdit, conflict: interfaces.IDocumentMergeConflict | null, ...args) {
+		const fileName = path.basename(editor.document.uri.fsPath);
+
+		// No conflict, command executed from command palette
+		if (!conflict) {
+			conflict = await this.findConflictContainingSelection(editor);
+
+			// Still failed to find conflict, warn the user and exit
+			if (!conflict) {
+				vscode.window.showWarningMessage(localize('cursorNotInConflict', messages.cursorNotInConflict));
+				return;
+			}
+		}
+
+		let range = conflict.current.content;
+		const leftUri = editor.document.uri.with({
+			scheme: ContentProvider.scheme,
+			query: JSON.stringify(range)
+		});
+
+		const leftTitle = `Current changes`; // (Ln ${range.start.line}${!range.isSingleLine ? `-${range.end.line}` : ''})`;
+
+		range = conflict.incoming.content;
+		const rightUri = leftUri.with({ query: JSON.stringify(range) });
+
+		const rightTitle = `Incoming changes`; // (Ln${range.start.line}${!range.isSingleLine ? `-${range.end.line}` : ''})`;
+
+		const title = `${fileName}: ${leftTitle} \u2194 ${rightTitle}`;
```

Comment:
Maybe inline the two titles and use a placeholder `{0}` to localize like: https://github.com/Microsoft/vscode/blame/master/extensions/git/src/commands.ts#L754